src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/InvitationRepository.cs

```diff
@@ -31,6 +31,36 @@ public void UpdateCommPkgOnInvitations(string projectName, string commPkgNo, str
             commPkgsToUpdate.ForEach(cp => cp.Description = description);
         }
 
+        public void MoveCommPkg(string fromProject, string toProject, string commPkgNo, string description)
```

Comment:
just thinking "out loud". Two cases:
IPO has one commpkg, and commpkg is moved to a new project. Then project on IPO should change as well?
IPO has more that one commpkg, and one commpkg is moved to another project. This creates illegal scope. How should we handle this?

Reply:
Hah, that's true! .. this is a problem preservation also have!!
I don't have the answer ... I guess preservation will "crash" in such cases because code is written to always get unique Tags within a project

Reply:
Hmm.. IPO with one commm pkg when the comm pkg is moved. That should maybe change project.
The other scenario is not possible to handle in the current setup. 
But that said, I think both scenarios are very unrealistic. We discussed it earlier, and then we just decided to ignore it :) As I guess moving om comm pkgs. happens much earlier in the project than at the stage where IPOs are worked with.